src/pages/OfferNew/Introduction/Sidebar/DetailsModal/index.tsx

```diff
@@ -0,0 +1,425 @@
+  InputGroupDeleteButton,
+  InputGroupRow,
+} from 'new-components/inputs/index'
+import * as React from 'react'
+import { useTextKeys } from 'utils/hooks/useTextKeys'
+import { DetailInput } from './DetailInput'
+import { SupportSection } from './SupportSection'
+import {
+  getFieldSchema,
+  getInitialValues,
+  getValidationSchema,
+  isApartmentFieldSchema,
+  isHouseFieldSchema,
+} from './utils'
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  padding 4rem 5rem 1.5rem 5rem;
+  display: flex;
+  flex-direction: column;
+
+  @media (max-width: 900px) {
+    padding: 3rem 3rem 1.5rem 3rem;
+  }
+
+  @media (max-width: 800px) {
+    padding: 2rem 2rem 1.5rem 2rem;
+  }
+
+  @media (max-width: 500px) {
+    padding: 2rem 1rem 1.5rem 1rem;
+  }
+`
+
+const Headline = styled.div`
+  font-family: ${fonts.GEOMANIST};
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+  color: ${colorsV2.black};
+
+  @media (max-width: 600px) {
+    font-size: 2rem;
+    line-height: 3rem;
+  }
+`
+
+const Content = styled.div`
+  width: calc(100% + 2rem);
+  display: flex;
+  flex-direction: row;
+  margin: 0 -1rem;
+  margin-top: 2.5rem;
+
+  @media (max-width: 800px) {
+    width: 100%;
+    margin: 1.5rem 0 0 0;
+    flex-direction: column;
+  }
+`
+
+const ContentColumn = styled.div`
+  width: calc(50% - 2rem);
+  margin: 0 1rem;
+
+  @media (max-width: 800px) {
+    width: 100%;
+    margin: 0 0 2rem 0;
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+`
+
+const ContentColumnTitle = styled.div`
+  font-size: 1rem;
+  color: ${colorsV2.gray};
+  margin-bottom: 1rem;
+  padding-left: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 2rem;
+
+  @media (max-width: 800px) {
+    padding: 0;
+  }
+`
+
+const ContentColumnTitleButton = styled.button`
+  padding: 0.375rem 1rem;
+  font-size: 0.875rem;
+  color: ${colorsV2.gray};
+  border-radius: 24px;
+  border: 1px solid ${colorsV2.gray};
+  cursor: pointer;
+  transition: all 250ms;
+  background: none;
+
+  :focus {
+    outline: none;
+  }
+
+  :hover {
+    color: ${colorsV2.darkgray};
+    border-color: ${colorsV2.darkgray};
+  }
+`
+
+const Footer = styled.div`
+  margin-top: 2.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`
+
+const Warning = styled.div`
+  font-size: 0.75rem;
+  line-height: 1rem;
+  margin-top: 1rem;
+  color: ${colorsV2.darkgray};
+  text-align: center;
+`
+
+const Error = styled.div`
+  font-size: 0.75rem;
+  line-height: 1rem;
+  margin-top: 1rem;
+  color: ${colorsV2.coral500};
+  text-align: center;
+`
+
+const LoadingDimmer = styled.div<{ visible: boolean }>`
+  background: rgba(255, 255, 255, 0.5);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  transition: all 350ms;
+  opacity: ${(props) => (props.visible ? 1 : 0)};
+  visibility: ${(props) => (props.visible ? 'visible' : 'hidden')};
```

Comment:
does `opacity` have _any_ effect here? considering we're using `visible`/`hidden` also?